### PR TITLE
Merge me when this day ends

### PR DESCRIPTION
**Pull request by yangwao**
- #5608 
- #5581 
- #5607 

Low prio
---
- #5604
- #5601

### CODE REVIEW

File: components/collection/activity/ownerInsightsTabs/FlipperTab.vue [hunk 1]
```diff
@@ -86,22 +94,24 @@ const toggleNFTDetails = (flipperId: string) => {
...
+    offset.value += 4
+  }
+})
+
+const displayedFlippers = computed(() => flippers.value.slice(0, offset.value))
 // map of flipper id to bolean, is the NFT details section of that flipper open or nor
 // {id0: false, id1: true, id3: false, ...}
 const isFlipperMoreNFTSectionOpen = flippers.value.reduce(
-  (accumelator, [holderId, _]) => ({
+  (accumelator, [flipperId, _]) => ({
```
Comment (vikiival) [inline components/collection/activity/ownerInsightsTabs/FlipperTab.vue:108]: lol typo here

Comment (daiagi) [reply components/collection/activity/ownerInsightsTabs/FlipperTab.vue:108]: 👀  where?

**Comment by yangwao** [comment]
when it builds I'm with release it tho 👀

File: components/collection/activity/ownerInsightsTabs/FlipperTab.vue (hunk 1 above)
Comment (vikiival) [reply components/collection/activity/ownerInsightsTabs/FlipperTab.vue:108]: 71dd9506f0bef088880eac46f0bfd2e77583a70f

Comment (vikiival) [reply components/collection/activity/ownerInsightsTabs/FlipperTab.vue:108]: > accumelator

Screenshot 2023-04-12 at 13 28 56

**Pull request merged**